packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/FiatInput/index.tsx

```diff
@@ -25,25 +26,30 @@ const FiatInput = () => {
         trigger,
         updateSendCryptoValue,
         setValue,
+        getValues,
     } = useCoinmarketExchangeFormContext();
 
-    const error = errors.outputs && errors.outputs[0] ? errors.outputs[0].fiat : undefined;
+    const amountError = errors.outputs && errors.outputs[0] ? errors.outputs[0].amount : undefined;
+    const fiatError = errors.outputs && errors.outputs[0] ? errors.outputs[0].fiat : undefined;
 
+    const { outputs } = getValues();
+    const output = outputs && outputs[0];
+    const fiat = output?.fiat;
     return (
         <StyledInput
             onFocus={() => {
                 trigger([FIAT_INPUT]);
             }}
             onChange={event => {
                 setValue('setMaxOutputId', undefined);
-                if (error) {
+                if (fiatError) {
                     setValue(CRYPTO_INPUT, '');
                 } else {
                     updateSendCryptoValue(event.target.value, network.decimals);
                     clearErrors(FIAT_INPUT);
                 }
             }}
-            state={error ? 'error' : undefined}
+            state={getInputState(fiatError ?? amountError, fiat)}
```

Comment:
bit more precise for the case when `fiatError` is defined but falsy and `amountError` is truthy
```suggestion
            state={getInputState(fiatError || amountError, fiat)}
```

Reply:
Done.